### ISSUE TITLE
Don't delete the authorization_* tables

### ISSUE THREAD

**Issue by punkstar** [meanbee/magedbm2#17]
If you delete these tables then you a `No Administrators role was found, data fixture needs to be run` error on running `admin:user:create` with magerun.

**Comment by punkstar** [comment]
In the meantime you can add the following to any of your config files:

```
table-groups:
  - id: admin
    description: Admin tables
    tables: '' # Don't remove the admin_* tables
```

**Comment by punkstar** [comment]
Fixed by #28.
